WalletWasabi/Blockchain/Analysis/FeesEstimation/AllFeeEstimate.cs

```diff
@@ -22,7 +22,7 @@ public class AllFeeEstimate : IEquatable<AllFeeEstimate>
 		public bool IsAccurate { get; private set; }
 
 		/// <summary>
-		/// int: fee target, decimal: satoshi/vByte
+		/// Int: fee target, int: satoshi/vByte
 		/// </summary>
```

Comment:
The capitalization of `int` is lower case.

Reply:
How about: `/// Fee estimations. int: fee target, int: satoshi/vByte`

Reply:
I think that's not good. This property is in the "AllFeeEstimate" class and it's called "Estimations". There's already a name duplication, your comment would introduce a 3rd one.

Reply:
I really don't know what to do with this one.
Is the following better? `/// Estimations. int: fee target, int: satoshi/vByte`

Reply:
Dictionary<int: fee target, int: satoshi/vByte>

Reply:
@molnard's suggestion makes the comment more expressive. ACK.